crates/stc_ts_type_checker/tests/conformance/controlFlow/dependentDestructuredVariables.stats.rust-debug

```diff
@@ -1,6 +1,6 @@
 Stats {
     required_error: 2,
     matched_error: 0,
-    extra_error: 23,
+    extra_error: 35,
```

Comment:
I think this is fine (https://github.com/dudykr/stc/issues/552)

Reply:
But, increase extra_error in DestructureVariables

So we need to solve the https://github.com/dudykr/stc/issues/552 issue first.

Reply:
But I decided to focus on making inference correct, not the numbers

Reply:
Yes, then we have to decide on the option name now.